pyccel/codegen/printing/ccode.py

```diff
@@ -1246,7 +1248,8 @@ def _print_Slice(self, expr):
         start = self._print(expr.start)
         stop = self._print(expr.stop)
         step = self._print(expr.step)
-        return 'new_slice({}, {}, {})'.format(start, stop, step)
+        slice_type = self._print(expr.slice_type)
+        return 'new_slice({}, {}, {}, {})'.format(start, stop, step, slice_type)
```

Comment:
Not sure why codacy doesn't care currently but this should probably be an f-string

Reply:
https://github.com/pyccel/pyccel/pull/1299/commits/ee8cf11658638a9af0b93edc9f4abc71323b9f89